### PR TITLE
Inline returns in arrow functions. NFC

### PR DESCRIPTION
**Pull request by RReverser**
Done with ast-grep + minimal formatting tweaks.

Commands:

1. `sg -p '($$$ARGS) => { return {$$$OBJ}; }' -r '($$$ARGS) => ({$$$OBJ})' -l js -i` to handle objects as they must be wrapped into parentheses
2. `sg -p '($$$ARGS) => { return $EXPR; }' -r '($$$ARGS) => $EXPR' -l js -i` for everything else.

Might need another pair of eyes for broken code but hopefully CI will catch it.

### CODE REVIEW

File: src/embind/embind.js [hunk 1]
```diff
@@ -1191,10 +1187,7 @@ var LibraryEmbind = {
         var setter = field.setter;
         var setterContext = field.setterContext;
         fields[fieldName] = {
-          read: (ptr) => {
-            return getterReturnType['fromWireType'](
-                getter(getterContext, ptr));
-          },
+          read: (ptr) => getterReturnType['fromWireType'](getter(getterContext, ptr)),
           write: (ptr, o) => {
```
Comment (sbc100) [inline src/embind/embind.js:1191]: I think these two might make more sense using method syntax?

Comment (RReverser) [reply src/embind/embind.js:1191]: Idk, maybe just splitting into lines is enough?

**Comment by RReverser** [comment]
> Can you run `test/runner other.*code_size* other.*metadce* --rebase` to see if there are any effects of the code size of simple programs?

Hm, I wouldn't expect any at least after minifier run, but worth checking. I'm away from computer now, could you run it perhaps?

**Comment by sbc100** [comment]
Sure I can run a rebase after landing

**Pull request merged**